### PR TITLE
Issue #3189507 by agami4: Fix alignment for the hero banner on the sky theme

### PR DESCRIPTION
**Pull request by agami4**
## Problem
The elements of the bottom on the hero banner have a wrong alignment.

## Solution
Add updates for the elements of the bottom on the hero banner 

## Issue tracker
https://getopensocial.atlassian.net/browse/YANG-4357
https://www.drupal.org/project/social/issues/3189507

## How to test
*For example*
- [ ] Go/create an idea and check elements of the bottom on the hero banner

## Screenshots
 before:
idea-bug

after:
idea-fix

## Release notes
The elements of the bottom on the hero banner have a correct alignment on the sky theme.

## Change Record
-

## Translations
-

### CODE REVIEW

File: themes/socialblue/templates/node/node--hero--sky.html.twig [hunk 1]
```diff
@@ -169,45 +169,47 @@
 
           {% block metaengage %}
             <div class="meta-engage">
-              <div class="hero-footer__list-left">
-                {% if comment_field_status is not empty %}
-                  <a href="#section-comments" class="badge badge--large"
-                     title="{% trans %}Total amount of comments{% endtrans %}">
+              {% if (comment_field_status is not empty) and (views_count is not empty and views_label) and (content.like_and_dislike) %}
```
Comment (ribel) [inline themes/socialblue/templates/node/node--hero--sky.html.twig:172]: @agami4 With this change you remove comment and likes from all other existing heroes. So it needs to be fixed in other way - maybe you need to make changes only in challenge hero template.

image

**Review by ribel — CHANGES_REQUESTED** [review]
@agami4 Changes in the template needs to be fixed or reverted - see a comment.
Also for all distro PRs please create an issue on drupal.org.

File: themes/socialblue/templates/node/node--hero--sky.html.twig (hunk 1 above)
Comment (agami4) [reply themes/socialblue/templates/node/node--hero--sky.html.twig]: @ribel, I've checked some nodes and this condition is no needed. I removed condition

Comment (agami4) [reply themes/socialblue/templates/node/node--hero--sky.html.twig]: @ribel, I've created an issue on Drupal.org

**Review by ribel — APPROVED** [review]

**Comment by ribel** [comment]
🍒 picked to 9.x, was not able to do it for 8.x due to conflict.

**Pull request merged**